giskard/llm/loaders/prompt_injections.py

```diff
@@ -0,0 +1,97 @@
+import ast
+from typing import Optional
+import pandas as pd
+
+from ...datasets.base import Dataset
+from ..evaluators.string_matcher import StringMatcherConfig
+
+INJECTION_DATA_URL = "https://raw.githubusercontent.com/Giskard-AI/prompt-injections/main/prompt_injections.csv"
+GISKARD_META_URL = "https://raw.githubusercontent.com/Giskard-AI/prompt-injections/main/giskard_meta_data.csv"
+
+
+def _check_matching_dfs_len(df1, df2):
+    if len(df1) != len(df2):
+        raise ValueError(
+            f"{__name__}: {INJECTION_DATA_URL} and {GISKARD_META_URL} should "
+            "have the same length and should be a one-to-one mapping of each other."
+        )
+
+
+def _check_meta_df_requirements(df):
+    if "expected_strings" not in df.columns:
+        raise ValueError(f"{__name__}: expected_strings are needed for the evaluation.")
+
+    if df.expected_strings.isnull().values.any():
+        raise ValueError(f"{__name__}: expected_strings column cannot have any NaN values.")
+    df.expected_strings = df.expected_strings.apply(ast.literal_eval)
```

Comment:
This validation belongs to the prompt-injection repo. I don't think it's needed here, adds extra code for no better reliability (execution would fail with non user-friendly error in any case).

Reply:
done